spec.md

```diff
@@ -121,6 +121,8 @@ A public module has a stable interface allowing it to be instantiated by *other*
 A public module will always be physically present in a compiled circuit.
 In effect, public modules are the exported identifiers of a circuit.
 
+The main module of a circuit must be public.
```

Comment:
This language would be better placed where the main module is discussed, after line 79:

``` diff
Each FIRRTL circuit must have exactly one *main module*.
The main module is a module that has the same name as the FIRRTL circuit.
+ The main module must be public (see [@sec:public-modules]).
```

Reply:
Good call, updated thanks!